test/integration/util/util.go

```diff
@@ -80,6 +81,11 @@ func runCommand(command string, commandPath string) (stdOut string, stdErr strin
 	return
 }
 
+func WaitForNSeconds(seconds int) error {
+	time.Sleep(time.Duration(seconds) * time.Second)
+	return nil
```

Comment:
return nil... while the function has error as return value.
why have a value in the first place? or just place the sleep inline instead.

Reply:
If I am not mistaken, Godog requires return value of type error. Will check that.

Reply:
From what I see, it is not godog calling this function (as part of a step definition), so I do not think this is necessary.

Reply:
@agajdosi can please fix this? the `return nil` is not required and error return type is not required.

Reply:
With no return value:

```go
func WaitForNSeconds(seconds int) {
	time.Sleep(time.Duration(seconds) * time.Second)
}
````

the step

```go
	s.Step(`^user (?:waits|waited) "(\d+)" seconds?$`,
		util.WaitForNSeconds)
```
throws an error when used in a feature:

```
Test run using Boot2Docker iso image.
panic: expected handler to return only one value, but it has: 0

goroutine 1 [running]:
github.com/minishift/minishift/vendor/github.com/DATA-DOG/godog.(*Suite).Step(0xc4203219e0, 0x9acfc0, 0xc4204476a0, 0x9ab5c0, 0xaa2078)
        /home/agajdosi/golang/src/github.com/minishift/minishift/vendor/github.com/DATA-DOG/godog/suite.go:103 +0x842
github.com/minishift/minishift/test/integration.FeatureContext(0xc4203219e0)
        /home/agajdosi/golang/src/github.com/minishift/minishift/test/integration/integration_test.go:301 +0x1da7
github.com/minishift/minishift/test/integration.TestMain.func1(0xc4203219e0)
        /home/agajdosi/golang/src/github.com/minishift/minishift/test/integration/integration_test.go:105 +0x2b
github.com/minishift/minishift/vendor/github.com/DATA-DOG/godog.(*runner).run(0xc42034a980, 0x14)
        /home/agajdosi/golang/src/github.com/minishift/minishift/vendor/github.com/DATA-DOG/godog/run.go:75 +0xaa
github.com/minishift/minishift/vendor/github.com/DATA-DOG/godog.RunWithOptions(0xa7c7e9, 0x9, 0xaa1ee8, 0x0, 0x0, 0x0, 0xc420300210, 0x10, 0xa79027, 0x6, ...)
        /home/agajdosi/golang/src/github.com/minishift/minishift/vendor/github.com/DATA-DOG/godog/run.go:167 +0x83b
github.com/minishift/minishift/test/integration.TestMain(0xc420487f20)
        /home/agajdosi/golang/src/github.com/minishift/minishift/test/integration/integration_test.go:113 +0x30f
main.main()
        github.com/minishift/minishift/test/integration/_test/_testmain.go:38 +0xf7
exit status 2
FAIL    github.com/minishift/minishift/test/integration 0.058s
make: *** [Makefile:206: integration] Error 1
```

Still I can remove the function as it is not needed, just wanted to add a step which could be practical for debugging purposes. Let me know please @LalatenduMohanty @gbraad

Reply:
can be removed, as it does not provide any additional debugging option